quizApp/schema.sql

```diff
@@ -0,0 +1,11 @@
+create table questions(
+    id serial primary key,
+    question text
+);
+
+create table answers(
+    id serial primary key,
+    answer text,
+    isCorrect boolean,
+    question_id integer references questions(id)
+);
```

Comment:
Nice.

Now, if you took this further, by adding a `students` table and an `attempts` table (to hold each students' answers for each question), what might those look like?